content/en/integrations/guide/powershell-command-to-install-azure-datadog-extension.md

```diff
@@ -27,7 +27,7 @@ To run the Datadog Agent in your Azure instances as an extension, use the comman
 {{% tab "Windows" %}}
 
 {{< code-block lang="powershell" >}}
-Set-AzVMExtension -Name "DatadogAgent" -Publisher "Datadog.Agent" -Type "DatadogWindowsAgent" -TypeHandlerVersion "7.0" -Settings @{"site" = "<SITE_PARAMETER>"; "agentVersion" = "latest"} -ProtectedSettings @{"api_key" = "<DATADOG_API_KEY>"} -DisableAutoUpgradeMinorVersion
+Set-AzVMExtension -Name "DatadogAgent" -Publisher "Datadog.Agent" -Type "DatadogWindowsAgent" -TypeHandlerVersion "7.0" -Settings @{"site" = "<SITE_PARAMETER>"; "agentVersion" = "latest"} -ProtectedSettings @{"api_key" = "<DATADOG_API_KEY>"} -DisableAutoUpgradeMinorVersion -VMName <VM_NAME> and -ResourceGroupName <RESOURCE_GROUP_NAME>
```

Comment:
Based on this [PowerShell doc](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_logical_operators?view=powershell-7.5#long-description), it looks like `and` should be `-and` here (and in a few other places of this update). Could you please confirm if this is the correct syntax?